src/InnerHit/NestedInnerHit.php

```diff
@@ -145,14 +147,14 @@ public function toArray()
     {
         $out = array_filter(
             [
-                'query' => $this->getQuery()->toArray(),
+                'query' => $this->getQuery() ? $this->getQuery()->toArray() : null,
```

Comment:
if `$this->getQuery()` will be null `array_filter` will remove it. So it's fine to have it like it was.